.circleci/config.yml

```diff
@@ -638,6 +641,7 @@ commands:
           master-cert-name: <<parameters.master-cert-name>>
           log-group-prefix: determined-ci
           retain-log-group: true
+          webhooks-base-url: ${WEBHOOKS_BASE_URL}
```

Comment:
I don't see this env variable set anywhere, did you mean `<<parameters.webhooks-base-url>>`?

Reply:
![Screen Shot 2022-11-15 at 4 29 29 PM](https://user-images.githubusercontent.com/103522725/202038572-7b570a24-20da-453d-af48-aca4a6dbf19e.png)

It is set in Circle CI but let me know if that doesnt sound or look correct.